homeassistant/components/hue/bridge.py

```diff
@@ -68,6 +68,10 @@ def host(self):
 
         hass.async_create_task(hass.config_entries.async_forward_entry_setup(
             self.config_entry, 'light'))
+        hass.async_create_task(hass.config_entries.async_forward_entry_setup(
```

Comment:
We could use a for loop here.

Reply:
I like the explicit calls to each. For loop doesn't add a lot I don't think, and means more indentation which hurts the readability with the 79 character limit.

Reply:
```py
forward_entry_setup = hass.config_entries.async_forward_entry_setup
for domain in ('binary_sensor', 'light', 'sensor'):
    hass.async_create_task(
        forward_entry_setup(self.config_entry, domain))
```